host_source.go

```diff
@@ -194,6 +196,20 @@ func (h *HostInfo) ConnectAddress() net.IP {
 	panic(fmt.Sprintf("no valid connect address for host: %v. Is your cluster configured correctly?", h))
 }
 
+func (h *HostInfo) getUntranslatedConnectAddress() net.IP {
```

Comment:
We do not start names with get as you can see here. Why not export it?

Reply:
Exported as `UntranslatedConnectAddress`